modules/onboarding/src/main/kotlin/edu/stanford/spezi/module/onboarding/consent/SignaturePad.kt

```diff
@@ -0,0 +1,132 @@
+package edu.stanford.spezi.module.onboarding.consent
+
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.Spacer
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.width
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.filled.Info
+import androidx.compose.material3.Button
+import androidx.compose.material3.Icon
+import androidx.compose.material3.Text
+import androidx.compose.material3.TextField
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Path
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.compose.ui.tooling.preview.PreviewParameterProvider
+import edu.stanford.spezi.core.design.theme.Spacings
+
+@OptIn(ExperimentalComposeUiApi::class)
+@Composable
+internal fun SignaturePad(
+    uiState: ConsentUiState,
+    onAction: (ConsentAction) -> Unit,
+) {
+    Column {
+        TextField(
+            value = uiState.firstName.value,
+            onValueChange = {
+                onAction(ConsentAction.TextFieldUpdate(it, TextFieldType.FIRST_NAME))
+            },
+            modifier = Modifier.fillMaxWidth(),
+            label = { Text("First Name") },
+            singleLine = true,
+            isError = uiState.firstName.error,
+            trailingIcon = { Icon(Icons.Filled.Info, contentDescription = "Information Icon") }
+        )
+        Spacer(modifier = Modifier.height(Spacings.small))
+        TextField(
+            value = uiState.lastName.value,
+            onValueChange = {
+                onAction(ConsentAction.TextFieldUpdate(it, TextFieldType.LAST_NAME))
+            },
+            modifier = Modifier.fillMaxWidth(),
+            label = { Text("Last Name") },
+            isError = uiState.lastName.error,
+            trailingIcon = {
+                Icon(
+                    Icons.Filled.Info,
+                    contentDescription = "Information Icon"
+                )
+            }
+        )
+
+        if (uiState.firstName.value.isNotBlank() && uiState.lastName.value.isNotBlank()) {
```

Comment:
@PSchmiedmayer 
that's a good point. i changed the order and used the FilledTonalButton instead of the regular button. here's an overview of the possible buttons, in case you think another one would fit better:
https://m3.material.io/components/buttons/overview

Reply:
Nice, that looks like a good fit. We should generally aim to follow a similar setup across the modules if possible, e.g., always use that style for secondary actions.